ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/patch-panel.j2

```diff
@@ -0,0 +1,21 @@
+{% if patch_panel is arista.avd.defined %}
+!
+patch panel
+{%     for patch in patch_panel.patches | arista.avd.natural_sort %}
+   patch {{ patch.name }}
+{%         if patch.enabled is arista.avd.defined(false) %}
+      shutdown
+{%         endif %}
+{%         for connector in patch.connectors %}
+{%             if connector.interface is arista.avd.defined %}
```

Comment:
missing check for connector.id.
Also see if you could refactor this to use a cli variable, so you only output cli once instead of once per combination.

Reply:
Check for connector.id added.